doc/releasenotes.html

```diff
@@ -59,7 +59,120 @@ <h2>**CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical
+</ul>
+
+<h2>Financial Models (2022.11.21)</h2>
+<ul>
+<li>Fix bug in Community Solar subscriber NPV calculation. (<a href="https://github.com/NREL/SAM/pull/1242">SAM 1242</a>, <a href="https://github.com/NREL/ssc/pull/929">SSC 929</a>)</li>
+<li>Change investment tax credit (ITC) inputs from number to array to allow it to be specified for one year or spread out over multiple years. (<a href="https://github.com/NREL/ssc/pull/913">SSC 913</a>)</li>
+</ul>
+
+<h2>Electricity Rates (2022.11.21)</h2>
+<ul>
+<li>For Utility Rate Database (URDB) downloads, apply look back months to billing demand inputs instead of listing as unused. (<a href="https://github.com/NREL/SAM/pull/1237">SAM 1237</a>)</li>
+<li>Fix rollover calculation for bill without system. (<a href="https://github.com/NREL/ssc/pull/941">SSC 941</a>)</li>
+</ul>
+
+<h2>Detailed Photovoltaic (2022.11.21)</h2>
+<ul>
+<li>Update module and inverter libraries. (<a href="https://github.com/NREL/SAM/pull/1269">SAM 1269</a>)</li>
+<li>Enhance bifacial model with electrical mismatch loss estimate and spatial albedo, and new Spatial tab on Results page. (<a href="https://github.com/NREL/SAM/pull/1152">SAM 1152</a>, <a href="https://github.com/NREL/ssc/pull/871">SSC 871</a>, <a href="https://github.com/NREL/SAM/pull/1199">SAM 1199</a>)</li>
+<li>Update P50/P90 simulations to PV Uncertainty with more sophisticated modeling of uncertainty factors. (<a href="https://github.com/NREL/SAM/pull/1076">SAM 1076</a>)</li>
+<li>Add option to input land area in acres/MWac and fix land area calculatoins. (<a href="https://github.com/NREL/SAM/pull/1195">SAM 1195</a>, <a href="https://github.com/NREL/SAM/pull/1166">SAM 1166</a>)</li>
+<li>Fix problem with terrain slope input enabling. (<a href="https://github.com/NREL/SAM/pull/1191">SAM 1191</a>)</li>
+<li>Improve error handling for different albedo input options. (<a href="https://github.com/NREL/SAM/pull/1180">SAM 1180</a>)</li>
+<li>Add warning in user interface for negative tilt angles. (<a href="https://github.com/NREL/SAM/pull/1171">SAM 1171</a>)</li>
+<li>Report latitude, longitude, elevation, and time zone from weather file header in simulation results. (<a href="https://github.com/NREL/ssc/pull/908">SSC 908</a>)</li>
+<li>New time series module temperature input to bypass internal calculations (SSC and LK script only, not available in SAM input pages). (<a href="https://github.com/NREL/ssc/pull/897">SSC 897</a>)</li>
+<li>Improve input checks in variable table for Software Development Kit and PySAM users. (<a href="https://github.com/NREL/ssc/pull/885">SSC 885</a>)</li>
+<li>Fix issue with AC wiring loss for subhourly simulations. (<a href="https://github.com/NREL/ssc/pull/879">SSC 879</a>)</li>
+</ul>
+
+<h2>PVWatts (2022.11.21)</h2>
```

Comment:
There was a significant bugfix to the PVWattsV8 inverter model. https://github.com/NREL/ssc/pull/872 I tagged this for the release just now.

Reply:
Thanks! I'll add that. I did see that there are a few PRs with no milestones, so we'll have to add those as we find them.